src/CodeGenerator/src/Generator/ResponseParser/ParserResult.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\CodeGenerator\Generator\ResponseParser;
+
+use Nette\PhpGenerator\Method;
+
+/**
+ * @internal
+ */
+class ParserResult
+{
+    /**
+     * @var string
+     */
+    private $body;
+
+    /**
+     * @var Method[]
+     */
+    private $extraMethods;
+
+    public function __construct(string $body, array $extraMethods = [])
```

Comment:
This will conflict with #677. 

Could you add a 2nd parameter `$usedClasses`?

Reply:
done